src/System.ComponentModel.TypeConverter/src/project.json

```diff
@@ -6,20 +6,29 @@
         "System.Collections.NonGeneric": "4.3.0-beta-24522-03",
         "System.Collections.Specialized": "4.3.0-beta-24522-03",
         "System.ComponentModel": "4.3.0-beta-24522-03",
+        "System.ComponentModel.EventBasedAsync": "4.3.0-beta-24522-03",
         "System.ComponentModel.Primitives": "4.3.0-beta-24522-03",
         "System.Diagnostics.Contracts": "4.3.0-beta-24522-03",
         "System.Diagnostics.Debug": "4.3.0-beta-24522-03",
         "System.Diagnostics.Tools": "4.3.0-beta-24522-03",
+        "System.Diagnostics.TraceSource": "4.3.0-beta-24522-03",
         "System.Globalization": "4.3.0-beta-24522-03",
+        "System.IO": "4.3.0-beta-24522-03",
+        "System.IO.FileSystem": "4.3.0-beta-24522-03",
         "System.Linq": "4.3.0-beta-24522-03",
+        "System.Net.WebClient": "4.3.0-beta-24522-03",
```

Comment:
We should file a follow-up item about breaking this dependency on WebClient, especially if it is tied to the license stuff which we aren't likely to fully support anyway.